.tekton/kn-plugin-func-func-util-115-pull-request.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift-knative/kn-plugin-func?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "release-v1.15" 
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: serverless-operator-135
+    appstudio.openshift.io/component: kn-plugin-func-func-util-115
+    pipelines.appstudio.openshift.io/type: build
+  name: kn-plugin-func-func-util-115-on-pull-request
+  namespace: ocp-serverless-tenant
+spec:
+  params:
+    - name: dockerfile
+      value: openshift/ci-operator/knative-images/func-util/Dockerfile
+    - name: build-args
+      value: [ VERSION=1.35.0,  GO_BUILDER=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.22,  GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal, ]
+    - name: git-url
+      value: '{{source_url}}'
+    - name: hermetic
+      value: "true"
+    - name: image-expires-after
+      value: 5d
+    - name: output-image
+      value: quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/kn-plugin-func-func-util:on-pr-{{revision}}
+    - name: revision
+      value: '{{revision}}'
+    - name: prefetch-input
+      value: "[{"path":".","type":"gomod"}]"
```

Comment:
/hold

This needs to include RPMs locking and it's not included because the generator wasn't invoked again

Reply:
On my list. But the pipelines are not setup yet anyhow

Reply:
@creydr @matejvasek how do you regenerate Dockerfiles ? is there a Makefile target we can invoke with https://github.com/openshift-knative/hack/actions/workflows/run-command-repository.yaml ?

Reply:
I am not aware of a Makefile target, but https://github.com/openshift-knative/kn-plugin-func/blob/release-v1.15/openshift/scripts/generate-dockerfiles.sh is the script to create the Dockerfiles (will be adjusted in #921 a bit)

Reply:
That action only runs a Makefile target